_data/baic/schedule/fall22.yml

```diff
@@ -0,0 +1,50 @@
+- abstract: "Biweekly Antiracism and Inclusion Conversations Joint Kick-off (Staff, Faculty and Grad Students)"
+  date: 2022-09-09
+  summary: Coming soon
+
+- abstract: "The Journey to Inclusive Cultures 101"
+  date: 2022-09-23
+  time: ""
+  summary: |
+    The Journey to Inclusive Cultures (JIC) 101, First of a three-part course. This foundational 90 minute workshop is designed to provide participants with shared definitions of core concepts in DEI work. We also provide context for how and why DEI conversations became commonplace in our current social context, and we ask participants to identify where they are currently on their DEI journey. Offered by the Inclusive Culture Team. Speakers will be Montez Butts, MA and Anthony C. Siracusa, Ph.D.
```

Comment:
Is registration required?

Reply:
All these events are held as part of BAIC so the answer should be know. 
I can add a TODO to update it when I get confirmation,